packages/inter-protocol/test/psm/gov-replace-committee.js

```diff
@@ -26,15 +24,7 @@ const { values } = Object;
 /** @type {<X, Y>(xs: X[], ys: Y[]) => [X, Y][]} */
 const zip = (xs, ys) => xs.map((x, i) => [x, ys[i]]);
 
-/**
- * @param {ERef<import('@agoric/vats').NameAdmin>} nameAdmin
```

Comment:
Rather than removing all the type declarations, just change the `import(...)` syntax to `@import`. That is:

```js
/**
 * @import { NameAdmin } from '@agoric/vats';
 */
```

and then:

```js
/**
 * @param { NameAdmin } nameAdmin
 */
```